README

```diff
@@ -193,6 +193,13 @@ REQUIREMENTS:
         under all circumstances. In fact, systemd-hostnamed will warn
         if nss-myhostname is not installed.
 
+        nss-systemd must be enabled on systemd systems, as that's required for
+        DynamicUser= to work. Note that we ship services out-of-the-box that
+        make use of it now, hence nss-systemd is not an option.
+
+        Note that the build prefix for systemd must be /usr. Changing it to
+        anything else is not supported and will break.
```

Comment:
I think this has potential for confusion: it *sounds* like split-usr is not supported anymore, and I'd rather avoid the brouhaha this could cause on the internets. Maybe it'd be better to just explicitly enumerate possible options. I looked at meson.build, and it seems that we have too many options there. I think we could just kill rootprefixdir= (and automatically derive it from split-usr=). So maybe just say that only "-Dsplit-usr=false (which is the default and does not need to be specified) is the recommended setting, and -Dsplit-usr=true should be used on systems which have /usr on a separate partition.".